src/lib/components/Accordion/Item.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+    export let open: boolean = false;
+    export let title: string = '';
```

Comment:
I would make this required 👍🏻 
```suggestion
    export let title: string;
```